core/src/util/usFramework.cpp

```diff
@@ -66,8 +66,8 @@ void Framework::Initialize(void)
   void* frameworkInit = NULL;
   std::memcpy(&frameworkInit, &initFncPtr, sizeof(void*));
   bundleInfo->location = BundleUtils::GetLibraryPath(frameworkInit);
-
-  d->coreBundleContext.bundleRegistry.RegisterSystemBundle(this, bundleInfo);
+  
+  d->coreBundleContext.bundleRegistry.RegisterSystemBundle(std::static_pointer_cast<Framework>(this->shared_from_this()), bundleInfo);
```

Comment:
Right, removing ```this->``` should be functionally the same. I'll remove ```this->``` from here and in usBundle.cpp